src/test/java/seedu/address/model/ModelTest.java

```diff
@@ -1,20 +1,28 @@
 package seedu.address.model;
 
-import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.*;
 
 import java.util.Arrays;
 import java.util.Collections;
 
 import org.junit.Before;
+
 import org.junit.Rule;
 import org.junit.Test;
 import org.junit.rules.ExpectedException;
 
+import junit.framework.Assert;
+
 import seedu.address.commons.exceptions.IllegalValueException;
+import seedu.address.logic.commands.ClearCommand;
+import seedu.address.logic.commands.Command;
+import seedu.address.logic.commands.CommandResult;
+import seedu.address.model.ModelManager.HeadAtBoundaryException;
 import seedu.address.model.task.TypicalDeadlineTasks;
 import seedu.address.model.task.TypicalEventTasks;
 import seedu.address.model.task.TypicalFloatingTasks;
 
+@SuppressWarnings("deprecation")
```

Comment:
Maybe replace all the `Assert.fail()` with `assert false : "Should not be called"`